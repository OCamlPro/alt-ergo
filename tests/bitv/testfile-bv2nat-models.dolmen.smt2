(set-logic ALL)
(set-option :produce-models true)
(declare-const x (_ BitVec 4))
(assert (= (bv2nat x) 7))
(check-sat)
(get-model)
