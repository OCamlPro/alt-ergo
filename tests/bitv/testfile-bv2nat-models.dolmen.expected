
unknown
(
  (define-fun x () (_ BitVec 4) #x7)
)
